src/home/swap-container/components/swap/index.tsx

```diff
@@ -217,6 +214,19 @@ const Swap = ({
   const shouldEnableYield =
     yieldEnabled && (fromCanHaveYield || toCanHaveYield) && isAtLeastAWeek && hasEnoughUsdForYield;
 
+  React.useEffect(() => {
+    if (!from) return;
+    setRate(
```

Comment:
I think some of this checks will set rate to false or undefined when they fail, and we are only checking if `rate !== ''` in line 202. if this is so, we might send an unsupported value to parseUnits